.github/workflows/ci-dockers-pytorch.yml

```diff
@@ -8,7 +8,7 @@ on:
     types: [opened, reopened, ready_for_review, synchronize]  # added `ready_for_review` since draft is skipped
     paths:
       - ".actions/**"
-      - ".github/workflows/ci-pytorch-dockers.yml"
+      - ".github/workflows/ci-dockers-pytorch.yml"
```

Comment:
@Borda this change needs to be done to the checkgroup config too.

The paths and checkgroup config have to be the same

Reply:
sorry, fixing it in #16984